.github/workflows/reusable-test.yml

```
@@ -62,9 +62,18 @@ jobs:
         run: |
           ./rippled --version | grep libvoidstar
 
-      - name: Test the binary
+      - name: Test the rippled binary
```

```suggestion
      - name: Run the embedded tests
```

Since `rippled` will be renamed soon, and there's only one binary produced anyway, adding the name of the binary is superfluous.

For contrast with the separate tests, I'd suggest use of the word `embedded` or `built-in`.